red_october/red_october.py

```diff
@@ -350,16 +350,17 @@ def call(self, endpoint, method='POST', params=None, data=None):
                 success.
         """
         endpoint = '%s/%s' % (self.uri_base, endpoint)
-        if data:
-            data.update({
-                'Name': self.name,
-                'Password': self.password,
-            })
+        if data is None:
```

Comment:
Derppppp! I totally forgot about that. The worse part is, I think this bit one of us in the butt with a project way back when.